app/models/form526_submission.rb

```diff
@@ -373,6 +374,8 @@ def submit_form_8940
 
   def submit_flashes
     user = User.find(user_uuid)
+    # Note that the User record is stored in Redis temporarily, for 30 minutes: User.redis_namespace_ttl => 1800
+    # If this method runs after the TTL, then there is a possibility the flashes will not be applied
     BGS::FlashUpdater.perform_async(id) if user && Flipper.enabled?(:disability_compensation_flashes, user)
```

Comment:
@michelpmcdonald Do you agree with this code comment?

Reply:
@annaswims (since you added this method in #7732) When the job retries several times, there's a possibility that this method is executed but the `user` record is no longer available in Redis. Do we still need to check the Flipper flag for particular users?

Reply:
@annaswims sadly is long gone....

Since you asked(I'm not personally too picky about comments), not in love with the comment, I think the code without the comment already clearly shows that flashes may not be applied if user is nil, and following through to the User object, it's pretty obvious that it's a reddis backed model.  Also, since the ttl is a setting, the actual value(1800) should not, IMHO, be enshrined in a comment...

I do think this code might be able to be improved, I'm unsure of the implications of not sending a flash, but since this code is now behind(time wise) the sidekiq fasttrack job, we now have more exposure to the Redis User model being ejected from the cache.  To harden this up, I'd look to see what info "flash" is using from user, and see if that info can be pulled from the auth_headers instead, but that is not part of the scope of this ticket...

Reply:
Since this looked like a bug, I wanted to make the behavior explicit in the comments. Also it took me a good amount of time to connect the dots, so a reminder comment seemed useful. I can reword them so the configuration values are not hard-coded.

The implication of not sending a flash is the claim is not prioritized like it should be.